opensrp-chw-core/src/main/res/layout-w900dp/activity_base.xml

```diff
@@ -8,400 +8,6 @@
-                    android:layout_height="wrap_content">
-
-                    <ImageView
-                        android:id="@+id/ivIcon"
-                        android:layout_width="40dp"
-                        android:layout_height="40dp"
-                        android:layout_alignParentStart="true"
-                        android:layout_alignParentLeft="true"
-                        android:layout_alignParentTop="true"
-                        app:srcCompat="@mipmap/sidemenu_sync" />
-
-                    <ProgressBar
-                        android:id="@+id/pbSync"
-                        android:layout_width="40dp"
-                        android:layout_height="40dp"
-                        android:layout_alignParentStart="true"
-                        android:layout_alignParentTop="true"
-                        android:indeterminate="true" />
-
-                </RelativeLayout>
-
-                <TextView
-                    android:id="@+id/tvSync"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_centerVertical="true"
-                    android:layout_marginLeft="10dp"
-                    android:layout_marginRight="10dp"
-                    android:layout_toEndOf="@+id/rlIconSync"
-                    android:layout_toRightOf="@+id/rlIconSync"
-                    android:fontFamily="sans-serif"
-                    android:text="@string/nav_sync"
-                    android:textColor="@color/white"
-                    android:textSize="24sp" />
-
-                <TextView
-                    android:id="@+id/tvSyncTime"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_alignParentEnd="true"
-                    android:layout_alignParentRight="true"
-                    android:layout_centerVertical="true"
-                    android:fontFamily="sans-serif"
-                    android:text="@string/nav_last_sync_time"
-                    android:textColor="@color/white"
-                    android:textSize="18sp" />
-
-
-                <TextView
-                    android:id="@+id/tvSyncTimeTitle"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_centerVertical="true"
-                    android:layout_toStartOf="@+id/tvSyncTime"
-                    android:layout_toLeftOf="@+id/tvSyncTime"
-                    android:fontFamily="sans-serif"
-                    android:text="@string/nav_last"
-                    android:textColor="@color/white"
-                    android:textSize="18sp" />
-
-            </RelativeLayout>
-        </RelativeLayout>
-
-    </com.google.android.material.navigation.NavigationView>
+    <include layout="@layout/navigation_view" />
```

Comment:
@LZRS Why was this file piece removed? Doesn't this affect all other chw Flavors?